srcpkgs/astroid/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'astroid'
+pkgname=astroid
```

Comment:
version= directly below pkgname=
revision= directly below version=

build_style= directly below revision=
hostmakedepends= directly below build_style=
makedepends= directly below hostmakedepends=, cut the line at 80 columns and the new line should have a whitespace prefix